doc: Modify the it8xxx2_evb overview document

hunterhu3000 opened the pull request:
1. Remove the obsoleted board
2. Add HW rework of the new it8xxx2_evb
3. Add Wiring of the new it8xxx2_evb
4. Update the Ubuntu flash guide

Signed-off-by: Hu Zhenyu <zhenyu.hu@intel.com>

keith-zephyr commented on boards/riscv/it8xxx2_evb/doc/index.rst:
```diff
@@ -73,6 +75,41 @@ Other hardware features are not currently supported by Zephyr.
 The default configuration can be found in the
 :zephyr_file:`boards/riscv/it8xxx2_evb/it8xxx2_evb_defconfig` Kconfig file.
 
+HW reworks
```
Nit - spell out "hardware"

keith-zephyr commented on boards/riscv/it8xxx2_evb/doc/index.rst:
```diff
@@ -73,6 +75,41 @@ Other hardware features are not currently supported by Zephyr.
+HW reworks
+**********
+
+Before using the it8xxx2_evb, some HW rework is needed:
```
same here - hardware

keith-zephyr commented on boards/riscv/it8xxx2_evb/doc/index.rst:
```diff
@@ -196,7 +225,10 @@ Ubuntu
+
+   .. code-block:: console
+
+      $ sudo ~/itetool/ite -f build/zephyr/zephyr.bin
```
Can add a new step 1 that instructs the user to download and compile the IT81202 Linux Flash Tool?

hunterhu3000 replied:
The tar file contains a README file which instructs the user to compile and run the tool

hunterhu3000 commented:
@keith-zephyr Thanks for your comments. I will update the document.
@GTLin08 Is there a link in the ITE official website to download the itedlb4-linux version 106 or later? Thanks

Dino-Li commented:
> GTLin08 Is there a link in the ITE official website to download the itedlb4-linux version 106 or later? Thanks

@hunterhu3000 The flash tool can be downloaded from here:
https://www.ite.com.tw/uploads/product_download/itedlb4-linux-v106.tar.bz2

stephanosio reviewed (dismissed):
There are some very large images added in this PR.

Please reduce the resolution and/or quality of the images so that are more reasonably sized (at most 100KB per image).

hunterhu3000 commented:
> There are some very large images added in this PR.
> 
> Please reduce the resolution and/or quality of the images so that are more reasonably sized (at most 100KB per image).

OK, will update it

hunterhu3000 commented:
@Dino-Li @GTLin08  could you please review and comment? Thanks

fabiobaltieri commented:
Hey do we really want such an extensive list of rework instruction to be hosted on the main code repository? I can understand some basic board configuration (close/open a bridge, add/remove a jumper...) but this feels like something that should be part of some board documentation hosted on the vendor site. Imagine having to update these and add more revisions and variations. Wouldn't it make more sense to point at an external document with the rework instructions?

NagarajuGundimi commented:
Most of the changes(rework instructions) looks good to me.

gmarull reviewed (dismissed):
-1 to discuss whether rework instructions should be in Zephyr docs or in a vendor document.

hunterhu3000 commented:
@gmarull @fabiobaltieri The HW rework content/pictures are removed. Thanks to review.

fkokosinski commented on boards/riscv/it8xxx2_evb/doc/index.rst:
```diff
@@ -8,9 +8,11 @@ Overview
 And a highly integrated embedded controller with system functions.
-It is suitable for mobile system applications.
+It is suitable for mobile system applications. The picture below is
+the INTEL_IT81302_81202_Colay_MECC_V01 develop board (also known as
```
This name looks weird. Can we spell it without underscores and in a more more human friendly way? Also EVB suggest evaluation board, not development board.

hunterhu3000 replied:
OK, the name INTEL_IT81302_81202_Colay_MECC_V01 is printed on the board. We can call it IT81302 MECC board.

fkokosinski commented on boards/riscv/it8xxx2_evb/doc/index.rst:
```diff
@@ -73,6 +75,12 @@ Other hardware features are not currently supported by Zephyr.
+****************
+
+Before using the it8xxx2_evb, some hardware rework is needed. It is suggested
+to ask the board vendor for the rework guide.
```
IIRC, this is were the long and detailed HW rework guide was before. Is it available somewhere on the vendor site? If so, can we link it here?

hunterhu3000 replied:
@Dino-Li @GTLin08  is it possible to upload a rework document somewhere, eg on ITE web site?

Dino-Li replied:
@hunterhu3000 sure, we will get back to you once it is ready.

Dino-Li replied:
@hunterhu3000 the rework document has been uploaded here:
https://www.ite.com.tw/uploads/product_download/IT81302_MECC_Rework_Guide_0927.pdf

hunterhu3000 replied:
@fkokosinski all the change requests are updated. Thanks to review and +1

hunterhu3000 committed:
doc: Modify the it8xxx2_evb overview document

1. Remove the obsoleted board
2. Add HW rework of the new it8xxx2_evb
3. Add Wiring of the new it8xxx2_evb
4. Update the Ubuntu flash guide

Signed-off-by: Hu Zhenyu <zhenyu.hu@intel.com>
